src/mainwindow.cpp

```diff
@@ -977,7 +969,9 @@ void ConvertJsonToStringMap(const QJsonObject& valJson, const QStringList& field
         }
         else if (val.isString())
         {
-            nameValues[field] = "\"" + val.toString().replace("\n", " ").replace("\"", "\"\"") + "\"";
+            QString strVal = val.toString();
+            strVal.truncate(32000); // Excel cannot handle more than ~32K chars.
+            nameValues[field] = "\"" % strVal.replace("\n", " ").replace("\"", "\"\"") % "\"";
```

Comment:
Timeline.twb preserves (or well, escapes and then restores) newlines. But QueryInfo.twb takes them away and then it becomes difficult to read queries in this last workbook.